app/profile/manageTask/page.tsx

```diff
@@ -76,29 +76,22 @@ export default function ManageTask() {
   const emptyTasksData: boolean = tasksData.length === 0;

```

Comment:
Tutaj chyba dodanie ręcznie : boolean nic ni wnosi, bo TS potrafi sobie to sam wywnioskować. To samo w powyższym useState